src/object.d

```diff
@@ -3190,11 +3199,24 @@ void __ctfeWriteln(T...)(auto ref T values) { __ctfeWrite(values, "\n"); }
  * Create RTInfo for type T
  */
 
+template RTInfoImpl(size_t[] pointers)
+{
+    immutable size_t[pointers.length] data = pointers[];
+    immutable RTInfoImpl = data.ptr;
```

Comment:
> Might also make sense to inline the bits for small block pools, e.g.
> [M bytes of pointer bitmap, e.g. 16 for alignment] + (M * 8 / N) x [N bytes block].
> Can we come up with a good scheme for that?

Do you mean something similar to what I tried with https://github.com/dlang/druntime/pull/1144, but only for the pointer bitmap? 16 bytes pointer bitmap already covers 1024 bytes of memory (assuming 64-bit pointers), so it won't fit into the same cache line for most blocks. Being on the same page might have a small advantage, too.

I'm pondering whether having different types of small block pages could work, e.g. some with pointer bitmaps, some with BlkAttr.APPENDABLE or a combination of these. These might store information before the aligned block, removing space from the previous block (modulo page size). This would be similar to the size storage in `rt.lifetime`, but without having to touch the end of the allocated memory.